app/client/components/Task/Task.js

```diff
@@ -269,6 +269,19 @@ export default class Task extends Component {
       return <div className={`status ${status}`} key={run.id}><span>{run.status} &middot; { that.relativeTime(run.start_time)}</span></div>
     })
   }
+  calculateRunDuration(run) {
+    var duration;
+    if (run['start_time'] && run['finish_time']){
+      try {
+        duration = (moment(run['finish_time']) - moment(run['start_time']))/1000
+        duration = (duration + 1).toFixed(0) // round to integer
```

Comment:
`moment` can display it a bit nicer: http://momentjs.com/docs/#/durations/humanize/

Reply:
I guess we already use the humanized version after, here - we try to be precise with the exat duration?

Reply:
I just wanted to keep it compact to not overflow the area. But yeah, we'll at least need to show minutes if there are very long tasks.